code/perception/src/Position_Publisher_Node.py

```diff
@@ -163,7 +168,9 @@ def update_imu_data(self, data: Imu):
         # ---------------------------------------------------------------
         heading = (raw_heading - (math.pi / 2)) % (2 * math.pi) - math.pi
```

Comment:
It seems this transformation doesn't make sense anymore with the heading fix, because the yaw already is the angle from the x-axis with 0 rad when looking into the x-axis direction.
For your testing purposes you should not use this transformation anymore and directly use the yaw :+1: